.ruby-version

```diff
@@ -1 +1 @@
-2.3.1
+2.3.3
```

Comment:
There _are_ performance improvements in 2.4.0, true. If it doesn't require any code changes I'll bump :+1: